references/plugins/references.go

```diff
@@ -569,21 +565,23 @@ func (ref *ParseReference) parseParameters(paraValue cue.Value, paramKey string,
 						return fmt.Errorf("failed to got Map kind from %s", param.Name)
 					}
 				} else {
-					if err := ref.parseParameters(val, name, depth); err != nil {
+					if err := ref.parseParameters(val, name, depth+1); err != nil {
 						return err
 					}
 					param.PrintableType = fmt.Sprintf("[%s](#%s)", name, name)
 				}
 			case cue.ListKind:
 				elem, success := val.Elem()
 				if !success {
-					return fmt.Errorf("failed to get elements from %s", val)
+					// fail to get elements, use the value of ListKind to be the type
+					param.Type = val.Kind()
+					param.PrintableType = val.IncompleteKind().String()
+					break
```

Comment:
If broke, are they any message in the cli for users?

Reply:
no need, actually the type will directly display in the Website. Users will see the complex type if can't be parsed well.